tests/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+sudo apt-get update -y
+sudo apt-get install -y bats
```

Comment:
❓ / ⛏️ : According to the [official docuentation](https://bats-core.readthedocs.io/en/stable/tutorial.html) you can install Bats and at a specific version as a Git Submodule

```
$ git submodule add https://github.com/bats-core/bats-core.git test/bats
```

or even "better", run it as a docker container 😍 https://bats-core.readthedocs.io/en/stable/installation.html

I kind of preffer this way as it is more explicit and allow us to specify the version.

Reply:
Partly resolved with https://github.com/govirtuo/kube-ns-suspender/pull/43